src/connection/resources.js

```diff
@@ -12,6 +12,13 @@ export class AuthenticationService {
   /**
```

Comment:
This file is auto-generated by a tool that get's data from Swagger. We don't need to add the API functions manually.